app/logic/getUpcomingEvents.py

```diff
@@ -0,0 +1,27 @@
+from app.models.program import Program
+from app.models.interest import Interest
+from app.models.event import Event
+from app.models.user import User
+from datetime import date, datetime, time
+
+def getUpcomingEventsForUser(user):
+
+    interestedEvent = (Event.select(Event, Program, Interest)
+                            .join(Program)
+                            .join(Interest, on=(Event.program == Interest.program))
+                            .where(Interest.user == user)
+                            )
+    upcomingEvents = []
+    for event in interestedEvent.objects():
+        today = date.today()
+        times = datetime.now().time()
+        event_time = event.timeStart
+        #FIXME: Change event.description to event.eventName (or just add it to the dict). When the eventName column is filled

```

Comment:
Go ahead and resolve this FIXME now. I think event should have a `name` field in addition to `description`.

Reply:
Done!